api/api.yaml

```diff
@@ -121,6 +121,8 @@ definitions:
       search_placeholder_text:
         type: string
         description: Optional. What to show in the search box by default
+      enable_search:
```

Comment:
enable_search_values or enable_search_facet_values
(Right now we allow search on facet name and description)